### DISCUSSION TITLE
Feature Request: Dashboard List: add description

### DISCUSSION THREAD

**Discussion by Cosmoledo** [grafana/grafana#78726] [General]
Using the default/native [Dashboard List plugin](https://grafana.com/grafana/plugins/dashlist/) is cool. It does list all existing dashboards, so adding new ones is no hussle as you do not need to add a new entry somewhere else. It will appear automatically.
But if you have multiple dashboards and different people use them at irregular times, they might forget if one already exists or what the purpose of one might be. Adding the dashboards description would be really helpful in those cases, you could directly read what it is intended to. I think about a new switch in the properties to display them.

Example of current behavior (Grafana Enterprise v10.0.3 (eb8dd72637), Dashboard List v5.0.0):
![Bildschirmfoto 2023-11-28 um 11 20 03](https://github.com/grafana/grafana/assets/26510897/becb9e0a-d405-4a63-882d-1a8a777375b6)

Example of "improved" behavior:
![Bildschirmfoto 2023-11-28 um 11 30 00](https://github.com/grafana/grafana/assets/26510897/2f7b2444-96ea-4608-948d-43b1f797af96)

Maybe you have better ideas of styling and positioning?

I also found those similar questions while searching, sadly they went nowhere:
- https://community.grafana.com/t/dashboard-description-is-it-used-anywhere/53273
- https://community.grafana.com/t/where-is-the-dashboard-description-actually-used/9119

As the native plugins are not open source (could not find anything), I also cannot fork/rewrite it myself...

**Comment by manimovassagh** [comment]
Just a heads up -- Grafana has since moved feature requests back to GitHub Issues (see the announcement in discussion #73424). So if you still want this, I would recommend opening a new issue in the grafana/grafana repo with the "type/feature-request" label.

That said, the Dashboard List panel plugin is actually open source -- it lives in the main Grafana repo at `public/app/plugins/panel/dashlist/`. If you wanted to take a crack at this yourself, the main file to look at is the panel component there. The dashboard search API already returns descriptions, so it would mostly be a matter of rendering them in the list items.

A potential workaround in the meantime: you can use a Text panel with HTML/Markdown to manually build a dashboard directory with descriptions and links. Not as dynamic as an auto-populating list, but it gives you full control over the presentation.
